### PR TITLE
Put value into quotes

### PR DESCRIPTION
**Pull request by koehlertimo**
It can be missunderstanding that "value" is a string!

Checklist:

- [ x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

**Comment by moT01** [comment]
Thanks for the PR @koehlertimo. I'm not convinced this is necessary since I'm pretty sure the value returned doesn't actually contain the quotes. I looked around a little and I see some strings in descriptions like this containing the quotes and some not, so I don't mind adding this - perhaps we could come to a consensus so we know what to do in other cases like this. I actually prefer not to have the quotes, but I'm fine with it either way. Alternate suggestion - we could maybe have it say `This code will still return the string <code>value</code> by default.` instead.

**Comment by RandellDawson** [comment]
> Alternate suggestion - we could maybe have it say This code will still return the string <code>value</code> by default. instead.

I like this idea better.  We have gone through much effort to change as many values surrounded by `code` tags to not be surrounded by quotes.  If this is something you think is needed, then I suggest opening an issue for discussion.  I personally do not want this PR merged, but if the consensus is to change all such instances in all applicable challenges, then I would be ok with it.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-arrow-functions-to-write-concise-anonymous-functions.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ When there is no function body, and only a return value, arrow function syntax a
 const myFunc = () => "value";
 ```
 
-This code will still return <code>value</code> by default.
+This code will still return <code>"value"</code> by default.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-arrow-functions-to-write-concise-anonymous-functions.md:35]:
```suggestion
This code will still return the string <code>value</code> by default.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
I'm on board with @moT01's suggestion, and agree that avoiding quotes is the correct approach.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-arrow-functions-to-write-concise-anonymous-functions.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-arrow-functions-to-write-concise-anonymous-functions.md:35]: @koehlertimo Can you commit @nhcarrigan's suggestion by click the `Commit suggestion` button if you are fine with this suggestion?  Thanks.

**Comment by RandellDawson** [comment]
@koehlertimo In an effort to save time, I went ahead and committed the change suggested by @nhcarrigan / @moT01.

@moT01 @nhcarrigan If you are OK, with these changes, go ahead and merge.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

**Review by moT01 — APPROVED** [review]
LGTM @koehlertimo 🎉 Congratulations on your first contribution to this freeCodeCamp repository 🎉

**Pull request merged**